### PR TITLE
[DO NOT MERGE] fix(docs): add Feb milestone

### PR DESCRIPTION
**Pull request by dhmlau**
Proposal for Feb milestone.
- Under the current plan, we will commit 44 points + one more task to be estimated. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/_feb-milestone_.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+- Migration guide between LB3 and LB4 MVP #453
+
+  - [ ] Which official LB3 components are not planned in LB4 #3956
+
+- Developer Experience
+
+  - [ ] Sugar decorators for Filter and Where parameters #1749
+
+- CI
```
Comment (bajtos) [inline docs/_feb-milestone_.md]: Can we please prioritize CI fixes higher? Either make them the first stretch goals to look at, or even move them to regular (committed) tasks.

File: docs/_feb-milestone_.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+### Stretch Goals
+
+- Migration guide between LB3 and LB4 MVP #453
+
+  - [ ] Which official LB3 components are not planned in LB4 #3956
+
+- Developer Experience
+
+  - [ ] Sugar decorators for Filter and Where parameters #1749
+
```
Comment (bajtos) [inline docs/_feb-milestone_.md]: I am proposing to add few bug fixes too, for example:

- Authorizer triggers twice #4351
- Belongs-To relation tries to resolve for foreign key NULL
- docs: API docs "edit this page" links to an invalid GitHub URL
- Log extension is not printing any logs #2173

File: docs/_feb-milestone_.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- (3) Add CrudRestApiBuilder to `@loopback/rest-crud` #3737
+
+- PR reviews:
+  - [ ] feat: adds @deprecated convenience decorator #4415
+  - [ ] feat: adds @tags convenience decorator #4416
+  - [ ] feat: adds utility launch for debugging a single test file #4432
+  - [ ] Add title to filter schemas #4355
+
+### Stretch Goals
+
```
Comment (bajtos) [inline docs/_feb-milestone_.md]: In on of our recent meetings, we agreed to regularly collect few "good first issue" tasks and offer them for the community contributors. I don't remember which venue we wanted to use for this, was it the milestone issue, the quarterly roadmap or one of our regular blog posts?

Comment (dhmlau) [reply docs/_feb-milestone_.md]: We could possibly do it in multiple places. I'm thinking milestone issue would be a good main one, and maybe blog posts too?

File: docs/_feb-milestone_.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  - [ ] Which official LB3 components are not planned in LB4 #3956
+
+- Developer Experience
+
+  - [ ] Sugar decorators for Filter and Where parameters #1749
+
+### "Help wanted" and "Good first issue"
+
+_Community contributors: Want to contribute but don't know where to start? Here
```
Comment (dhmlau) [inline docs/_feb-milestone_.md]: @bajtos, I think we discussed the two below.  Do you have other items to consider?

File: docs/_feb-milestone_.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+- Shopping app web site
+
+  - [ ] (2) Shopping App Website: Installation/setup documentation
+        https://github.com/strongloop/loopback4-example-shopping/issues/477
+
+- (3) Run shared tests from both v3 and v4 of juggler
+  https://github.com/strongloop/loopback-connector-db2/issues/133
+
+- (3) Add CrudRestApiBuilder to `@loopback/rest-crud` #3737
+
```
Comment (dhmlau) [inline docs/_feb-milestone_.md]: Would like to add some follow up tasks coming from the APIC/LB integration spike: https://github.com/strongloop/loopback-next/issues/4115#issuecomment-576845310.

File: docs/_feb-milestone_.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- PR reviews:
+  - [ ] feat: adds @deprecated convenience decorator #4415
+  - [ ] feat: adds @tags convenience decorator #4416
+  - [ ] feat: adds utility launch for debugging a single test file #4432
+  - [ ] Add title to filter schemas #4355
```
Comment (dougal83) [inline docs/_feb-milestone_.md]: FYI #4355 Merged.

**Comment by dhmlau** [comment]
With [APIC/LB integration spike](https://github.com/strongloop/loopback-next/issues/4115) and [authentication migration spike](https://github.com/strongloop/loopback-next/issues/3719) are done, I've added a few follow up tasks coming from the spikes.

File: docs/_feb-milestone_.md (hunk 2 above)
Comment (raymondfeng) [reply docs/_feb-milestone_.md]: #4351 is created to fix #4351

Comment (raymondfeng) [reply docs/_feb-milestone_.md]: I would like to land https://github.com/strongloop/loopback-next/pull/4117 soon.

Comment (dhmlau) [reply docs/_feb-milestone_.md]: - marked #4351 as working in progress
- added #4117 in PR review section.

**Comment by dhmlau** [comment]
Closing this PR as https://github.com/strongloop/loopback-next/issues/4543 is created.

**Pull request closed without merging**